papermill/preprocess.py

```diff
@@ -49,10 +33,9 @@ class PapermillExecutePreprocessor(ExecutePreprocessor):
     # Copyright (c) IPython Development Team.
     # Distributed under the terms of the Modified BSD License.
 
-    # TODO: Delete this wrapper when nbconvert allows for setting preprocessor
-    # hood in a more convienent manner
+    # TODO: Delete this wrapper and push shared changes to nbconvert
 
-    def preprocess(self, nb, resources):
+    def preprocess(self, engine_nb, resources):
```

Comment:
Perhaps preprocess.py could be moved to an nbconvert directory or renamed to make it more obvious that the term "preprocessor" is to be understood in the context of nbconvert. Without prior exposure to nbconvert, I spent a while wondering what the notebook was being preprocessed for.

Reply:
Yea the preprocessor name is really misleading in nbconvert. I can isolate the nbconvert names a bit more as you suggested.